content/en/news/releases/1.8.x/announcing-1.8/change-notes/index.md

```diff
@@ -0,0 +1,186 @@
+  ([Issue #25744](https://github.com/istio/istio/issues/25744))
+
+- **Added** support for `INSERT_FIRST`, `INSERT_BEFORE`, `INSERT_AFTER` insert operations for `HTTP_ROUTE` in `EnvoyFilter`  ([Issue #26692](https://github.com/istio/istio/issues/26692))
+
+- **Added** `REPLACE` operation for `EnvoyFilter`. `REPLACE` operation can replace the contents of a named filter with new contents. It is only valid for `HTTP_FILTER` and `NETWORK_FILTER`.
+  ([Issue #27425](https://github.com/istio/istio/issues/27425))
+
+- **Added** Istio resource status now includes observed generation
+  ([Issue #28003](https://github.com/istio/istio/issues/28003))
+
+- **Fixed** remove endpoints when the new labels in `WorkloadEntry` do not match the `workloadSelector` in `ServiceEntry`.
+  ([Issue #25678](https://github.com/istio/istio/issues/25678))
+
+- **Fixed** when a node has multiple IP addresses (e.g., a VM in the mesh expansion scenario),
+Istio Proxy will now bind `inbound` listeners to the first applicable address in the list
+(new behavior) rather than to the last one (former behavior).
+  ([Issue #28269](https://github.com/istio/istio/issues/28269))
+
+## Security
+
+- **Improved** Gateway certificates to be read and distributed from Istiod, rather than in the gateway pods.
+This reduces the permissions required in the gateways, improves performance, and will make certificate reading
+more flexible in the future. This change is fully backwards compatible with the old method, and requires no changes
+to your cluster. If required, it can be disabled by setting the `ISTIOD_ENABLE_SDS_SERVER=false`
+environment variable in Istiod.
+
+- **Improved** TLS configuration on sidecar server side inbound paths to enforce `TLSv2` version along with recommended cipher suites.
+             If this is not needed or creates problems with non Envoy clients, it can disabled by setting env variable `PILOT_SIDECAR_ENABLE_INBOUND_TLS_V2` to false.
+
+- **Updated** The `ipBlocks`/`notIpBlocks` fields of an `AuthorizationPolicy` now strictly refer to the source IP address of the IP packet as it arrives at the sidecar.  Prior to this release, if using the Proxy Protocol, then the `ipBlocks`/`notIpBlocks` would refer to the IP address determined by the Proxy Protocol.  Now the `remoteIpBlocks`/`notRemoteIpBlocks` fields must be used to refer to the client IP address from the Proxy Protocol.
+ ([reference](/docs/reference/config/security/authorization-policy/))([usage](/docs/ops/configuration/traffic-management/network-topologies/))([usage](/docs/tasks/security/authorization/authz-ingress/)) ([Issue #22341](https://github.com/istio/istio/issues/22341))
+
+- **Added** `AuthorizationPolicy` now supports nested JWT claims.
+  ([Issue #21340](https://github.com/istio/istio/issues/21340))
+
+- **Added** support for client side Envoy secure naming config when trust domain alias is used.
```

Comment:
Not sure if @incfly worked on this one, if you worked on this can we add a link to the github issue?

Reply:
https://github.com/istio/istio/pull/26185

the PR has linked for the design. we don't have an issue number.

Reply:
It's ok to add a PR link if there is no issue available for it. See [this](https://github.com/istio/istio.io/pull/8415/commits/fb1f312bec816296f92500205f2e485bf1381263).